app/Http/Requests/StoreAlbum.php

```diff
@@ -24,7 +24,16 @@ public function authorize()
     public function rules()
     {
         return [
-            //
+            'artists_id' => 'required|integer|exists:artists,id',
+            'title' => 'required|string|max:255',
+            'alt_title' => 'string|max:255',
+            'year' => 'required|integer',
```

Comment:
Given that we'll need to generate a UI component (dropdown menu) to present a range of years to be selected -- unless we want to allow a freeform text entry for some reason -- maybe we should use the same range here. Something like 1900-current year (or current year + 1, to accommodate staging a release prior to its actual release date)?

Reply:
How would you do `current date` to range via a validation rule ?

Reply:
I think it would be as simple as `'year' => 'required|integer|min:1900|max:' . (date(Y) + 1),`

Reply:
Oh I see what you did there .. Is this what you want me to change it to ?

Reply:
As long as you don't see any issue with doing that... I can't think of any reason it would be problematic.

Reply:
Aslo, should be sure to cast `date` to string ?

Like:
```php
'year' => 'required|integer|min:1900|max:' . (string)(date('Y') + 1),
```

Reply:
Yes, even better. This reminds me, we should establish this explicit casting a best-practice in the documentation, too.